operator/src/main/java/com/oracle/coherence/k8s/operator/CoherenceOperator.java

```diff
@@ -153,11 +153,10 @@ static void assertNamespaces(String[] asIncludedNamespaces, String[] asExcludedN
          */
         NamespaceProcessor(String sNamespace, String[] asIncludedNamespaces, String[] asExcludedNamespaces)
             {
-            m_sElasticsearchHost = ((System.getenv(ES_HOST) == null) ?
-                    "elasticsearch." + sNamespace + ".svc.cluster.local" : System.getenv(ES_HOST));
+            m_sElasticsearchHost = Env.get(ES_HOST, "elasticsearch." + sNamespace + ".svc.cluster.local");
+            m_sElasticsearchPort = Env.get(ES_PORT, DEFAULT_ES_PORT);
 
-            m_sElasticsearchPort = ((System.getenv(ES_PORT) == null) ? DEFAULT_ES_PORT :
-                    Integer.toString(Integer.parseInt(System.getenv(ES_PORT))));
+            assertInteger(m_sElasticsearchPort);
```

Comment:
This, combined with the actual implementation that simply calls `Integer.parseInt` seems dangerous...

We will throw `NumberFormatException` from here if the specified port is not a valid number. Is that exception handled somewhere higher up in the call chain or is it going to crash operator?

It seems a bit silly to crash just because someone misconfigured the port...

Reply:
The exception is handled somewhere higher up in the call chain: (line 82)
LOGGER.log(Level.SEVERE, "Cannot start watchers: " + t, t);
Note that the ES will still be installed but will not start properly.